src/types/duration.rs

```diff
@@ -0,0 +1,59 @@
+use napi::{bindgen_prelude::BigInt, Error, Status};
+use scylla::frame::value::CqlDuration;
+
+#[napi]
+pub struct DurationWrapper {
+  pub months: i32,
+  pub days: i32,
+  pub nanoseconds: i64,
+}
+
+#[napi]
+impl DurationWrapper {
+  #[napi]
+  pub fn new(months: i32, days: i32, ns_bigint: BigInt) -> napi::Result<Self> {
+    let ns = ns_bigint.get_i64();
+    if !ns.1 {
+      return Err(Error::new(
+        Status::GenericFailure,
+        "Invalid use: Should not happen?",
+      ));
+    }
```

Comment:
What can trigger this error? Please reflect the possible cause in the error message.